src/icon-manifest/supportedExtensions.ts

```diff
@@ -2546,6 +2546,12 @@ export const extensions: IFileCollection = {
       languages: [languages.t4],
       format: FileFormat.svg,
     },
+    {
+      icon: 'tt',
+      extensions: ['tt2', 'tt3'],
```

Comment:
`tt3` is already supported by the `languadeId` (https://github.com/samosad/vscode-template-toolkit/blob/master/package.json#L41)